applicationinsights.dev.json

```diff
@@ -24,7 +27,23 @@
               "matchType": "regexp"
             }
           ],
-          "percentage": 100
+          "percentage": 10
+        },
+        {
+          "telemetryType": "dependency",
```

Comment:
If you've gone to the latest gradle plugin then this shouldn't be required

Reply:
are you saying all of this JSON object or just "telemetryType": "dependency"?

Reply:
the sampling change higher up is required as otherwise it doesn't sample all requests.  This sampling override section isn't required as it was fixed in app insights 3.4.10

Reply:
will sort it